src/models/node.js

```diff
@@ -930,11 +978,26 @@ const Node = {
    */
 
   getTexts() {
+    return List(this._getTexts())
```

Comment:
Good question. There is a benchmark for `document.getTexts()`. On my machine this is the results:
- Array 52,000 ops/sec
- List 16,000 ops/sec

I would expect it to be faster to concat List (linked list) than Arrays (copied values)... So I don't know why.